### DISCUSSION TITLE
知末 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#147] [模板讨论]
知末 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by qukai518** [comment]
知末改成小程序签到了！请大佬更新一下！

**Reply by wjf0214** [reply]
我改了下，你可以试试。
没有多余的账号，不确定有没有用。
不行我再换另一种方式看看。目前用网站的cookie可以试试。有问题再及时反馈。

**Reply by wjf0214** [reply]
找人测了下。可以正常签到。
